comp/otelcol/otlp/components/processor/infraattributesprocessor/config.go

```diff
@@ -7,13 +7,17 @@ package infraattributesprocessor
 
 import (
 	"go.opentelemetry.io/collector/component"
+
+	"github.com/DataDog/datadog-agent/comp/core/tagger/types"
 )
 
 // Config defines configuration for processor.
 type Config struct {
 	Metrics MetricInfraAttributes `mapstructure:"metrics"`
 	Logs    LogInfraAttributes    `mapstructure:"logs"`
 	Traces  TraceInfraAttributes  `mapstructure:"traces"`
+
+	Cardinality types.TagCardinality `mapstructure:"cardinality"`
```

Comment:
My understanding is that cardinality is per-tag/attribute so I don't see how this config entry will specify cardinality for everything.

Reply:
After further discussions with @dineshg13, `Cardinality` is indeed a property of the agent/collector so this is correct. Resolving.